gitpod-io/gitpod
# Cell Disabled mode

## Description

Introduces a way for us to expire POV cells, controlled by the new `cell_disabled` feature flag.

## Related Issue(s)

Fixes ENT-794

## How to test

Visit the following preview env. You can also try flipping the feature flag mentioned above and see what it does.

https://ft-disable8a89f3dd88.preview.gitpod-dev.com/
